libs/application-generic/src/webhooks/usecases/send-webhook-message/send-webhook-message.usecase.ts

```diff
@@ -75,8 +91,6 @@ export class SendWebhookMessage {
         `Failed to send webhook ${command.eventType} for application ${appId}. Error: ${error.message}, Event ID: ${eventId}`,
         error.stack
       );
-
-      throw error;
```

Comment:
im not sure if we should throw the action after a webhook request failure, such behavior could cause unexpected behavior in the trigger engine. And if we want, we need to add exception handling support because right now it would look like this.

<img width="1168" height="821" alt="Image" src="https://github.com/user-attachments/assets/203b7642-0a17-43dc-b71e-ac44e8ceb618" />

cc @scopsy